specification/chaos/resource-manager/Microsoft.Chaos/preview/2022-10-01-preview/types/experiments.json

```diff
@@ -0,0 +1,812 @@
+          "description": "The name of the setting for the action.",
+          "type": "string",
+          "minLength": 1
+        },
+        "value": {
+          "description": "The value of the setting for the action.",
+          "type": "string",
+          "minLength": 1
+        }
+      },
+      "required": [
+        "key",
+        "value"
+      ],
+      "additionalProperties": false
+    },
+    "selector": {
+      "type": "object",
+      "description": "Model that represents a selector in the Experiment resource.",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Enum of the selector type.",
+          "enum": [
+            "Percent",
+            "Random",
+            "Tag",
+            "List"
+          ],
+          "x-ms-enum": {
+            "name": "SelectorType",
+            "modelAsString": false
+          }
+        },
+        "id": {
+          "type": "string",
+          "description": "String of the selector ID.",
+          "minLength": 1
+        },
+        "targets": {
+          "type": "array",
+          "description": "List of Target references.",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/targetReference"
+          }
+        },
+        "filter": {
+          "$ref": "#/definitions/filter"
+        }
+      },
+      "required": [
+        "type",
+        "id",
+        "targets"
+      ],
+      "additionalProperties": false
+    },
+    "filter": {
+      "type": "object",
+      "description": "Model that represents available filter types that can be applied to a targets list.",
+      "discriminator": "type",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Enum that discriminates between filter types. Currently only `Simple` type is supported.",
+          "enum": [
+            "Simple"
+          ],
+          "x-ms-enum": {
+            "name": "FilterType",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "simpleFilter": {
+      "type": "object",
+      "description": "Model that represents a simple target filter.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/filter"
+        }
+      ],
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/simpleFilterParameters"
+        }
+      },
+      "additionalProperties": false,
+      "x-ms-discriminator-value": "Simple"
+    },
+    "simpleFilterParameters": {
+      "description": "Model that represents the Experiment properties model.",
+      "type": "object",
+      "properties": {
+        "zones": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "String of the availability zone ID."
+          },
+          "description": "List of Azure availability zones to filter targets by.",
+          "minItems": 0
```

Comment:
Remove? Minimum items can never be less than zero.

Reply:
Removed.